.eslintrc.yml

```diff
@@ -1 +1,5 @@
 extends: 'eslint-config-cheminfo'
+parserOptions:
+  sourceType: module
+env:
+  jest: true
```

Comment:
you can remove the env parameter. It is now correctly configured in the cheminfo config